### DISCUSSION TITLE
CAN we stopppppp breaking every single one of my sites twice a week for like maybe a month..... Please

### DISCUSSION THREAD

**Discussion by sreidnoe** [elementor/elementor#23936] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

I have to spend an hour twice a week updating, then troubleshooting, then another half hour reading a thread, then when it hasn't been answered in the community support doesn't get back to us for FOUR days each time there is an issue because the breaks are coming faster than the fixes are..................... 

I just want ONE week where NOTHING is broken.... It has been for almost a month and a half now of nothing but fixing what NO ONE has an answer to... 

We have a few choices........... REDO ENTIRE websites and charge our customers for what you break..... OR have a security vulnerability by not performing the updates, OR GIVE our time away for FREE to convert the site. 

I have tried updating one of my sites 3 times now.... I have performed all of the fixes, nothing..... I have talked to support multiple times for this site for OTHER items that have broken.... THREE in the the last MONTH.... The down time is costing all of your developers who own their own small businesses TONS in labor alone.... 

### Describe the solution you'd like

Dude seriously.... I am trying to NOT have to go learn a new design system, but I've had enough of this.... For almost two months now we can't even perform ONE update without having to restore it or troubleshoot why its broken THIS TIME...  

### Describe alternatives you've considered

I am looking into Oxygen as well as other hosting and design elements.... I'd rather not have to switch, but this is now gotten to the point where I am done troubleshooting and being left for days in limbo with support.... 

### Additional context

At the end of writing this..... An hour and a half into trying to fix this site for the second time this week, not getting help from "Support" and overall done with this back and fourth..... I am finding another hosting solution, design creator, and getting the crap off this insane roller coaster..... You all should be ashamed of this... I am... I don't even know how to keep explaining this inferior performance to MY CLIENTS... They aren't your clients. They are your small business owners' clients... You are dragging our name in the mud right there with you.... It flipping  11:30.... This should have been an easy in and out, but nothing is anymore with this!

**Comment by toineenzo** [comment]
That's why you always make a backup or use a staging environment... Then there is no complaining fixing stuff after doing an update. You can just roll back or don't push to production. Also remember that not every problem is related to Elementor, but third party plugins can play a big role. So if you're using third party plugins, make sure they are of good quality and support.

**Comment by sreidnoe** [comment]
I do have backups. Rolling back doesn’t always fix it, and I have rolled
back, and Elementor still isn’t working properly. I have done all of the
disable all of my plugins, blah blah etc etc. Enter entry level updating
here. I’m not ignorant, nor do I pop off and ask questions before I have
tried every “fix” in here,  chats, community, and troubleshooting for
HOURS.

I’ve been at this 2 years and I’ve never had problems like I’ve had in the
last 2 months. Ever with multiple sites everywhere. Broken after update
with no conceivable reason, other than “it’s not our fault,” and we are
left out to dry then it turns out it’s a setting in Elementor that is very
much dependent on proper rendering.

Then when you do have an issue, support is so swamped that it’s four days
before they can even resolve or attempt to reach out. That says that there
are more issues than solutions and/or problem solvers.

I usually end up creating a work around before they even try to fix the
issue, but I’m over that. I have my client’s entire portfolio. I’m not just
doing the websites, I’m helping them run their entire business. I don’t
have the time to waste on something that has spent more time on the rack
than on the road.

The complaint is that it shouldn’t be this difficult to maintain a website
and that service for a product that we pay for should work more than it
doesn’t.

Half of my sites were grandfathered in with the $99/yr plan. It doesn’t
include staging. Just the backups. Whole. Point. Being.

If it were a car, it would be time to trade.

**Reply by toineenzo** [reply]
Alright, but how can we help you if you don't provide any system info? I don't even know what plugins you're using. Maybe I or someone else here can help you with the problems.

**Comment by sreidnoe** [comment]
Yeah no. I’ve done everything that anyone would reasonably know how to do
trouble shooting wise. The fact that I have just yesterday been contacted
by support last night at 11pm.

There is no earthly reason why I should have to be here as much as I have
been in the last two months.

I’ve been using it for two years. I’m not a non techie, I’m a developer. I
properly troubleshoot….. The plug-ins ARENT the problem. I’ve disabled them
all and the problem STILL persists.

I didn’t come here for sarcasm or someone talking to me like I don’t know
basic troubleshooting. I am 6 days in and FINALLY got someone from support
at 11pm central last night.

I’m over the last two months being NOTHING but headaches.

**Comment by sreidnoe** [comment]
I’m rebuilding the site with my intern to use NO Elementor elements and
then we are canceling that subscription. I’m migrating one by one.

I have other developers. We will just create an instance rather than using
Elementor and we will insert our templates right into the directories and
be done with it all together.
